src/Makefile

```diff
@@ -24,7 +24,18 @@ MULT_KERN = default
 DIAG_DUMMY = 
 
 #Include system-dependent variables
-include system.make
+#Get the system name automatically.
+SYSTEM = $(shell hostname -d | cut -d. -f1)
+ifneq ($(SYSTEM),)
+SYSTEM := .$(SYSTEM)
+endif
+SYSTEM_PATH = system/system$(SYSTEM).make
+ifneq ("$(wildcard $(SYSTEM_PATH))","")
+$(info Building on SYSTEM $(SYSTEM), using makefile $(SYSTEM_PATH))
```

Comment:
This is pretty minor, but since you've prefixed `SYSTEM` with a `.` it gets printed as `.<system name>` here